product_variant_supplierinfo/views/product_product_view.xml

```diff
@@ -7,24 +7,21 @@
     <record id="product_normal_form_view_sellers" model="ir.ui.view">
         <field name="name">product.product.form.sellers</field>
         <field name="model">product.product</field>
-        <field name="inherit_id" ref="product.product_normal_form_view"/>
+        <field name="inherit_id" ref="product.product_variant_easy_edit_view"/>
```

Comment:
Wouldn't it be better to add new fields in both views: `product_variant_easy_edit_view` and `product_normal_form_view`?
@pedrobaeza, @mourad-ehm, what's your point of view?